atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -2260,8 +2260,7 @@ private void handleKeyAlreadyExistsException(
                         + " because our locks timed out. startTs: " + getStartTimestamp() + ".  "
                         + getExpiredLocksErrorString(commitLocksToken, expiredLocks), ex);
             } else {
-                log.debug("Possible bug: Someone rolled back our transaction but"
-                        + " our locks were still valid; Atlas code is not allowed to do this, though others can.",
+                log.info("This transaction has been rolled back by someone else.",
                         SafeArg.of("immutableTimestampLock", immutableTimestampLock),
```

Comment:
This will break logging again: LeasedLockToken is not loggable by our logging infra. So either get rid of the args in this logline or figure out how to collect the stuff you need in a way that's loggable.

Reply:
For this to be useful, probably startTimestamp would be pretty useful as an arg?